src/main/java/codeu/controller/ActivityFeedServlet.java

```diff
@@ -0,0 +1,24 @@
+package codeu.controller;
+
+import codeu.model.data.Conversation;
+import codeu.model.data.User;
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.UserStore;
+import java.io.IOException;
+import java.time.Instant;
+import java.util.List;
+import java.util.UUID;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for the activity feed page. */
+public class ActivityFeedServlet extends HttpServlet{
+
+    @Override
```

Comment:
Use two spaces instead of four for indentation (to be consistent with the existing codebase).

Reply:
Done